dts/bindings/flash_controller/zephyr,disk-flash.yaml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: Emulates a flash using a disk
+
+compatible: "zephyr,disk-flash"
+
+include: base.yaml
+
+properties:
+  size:
+    type: int
+    description: flash capacity in bits
```

Comment:
Yes the emulated capacity. I adjusted the description accordingly. It was in bits since the jedec driver as well as the atmel did it that way. I rewrote it since it does not seem to be a convention.